### ISSUE TITLE
Module not compatible with InnoDB+utf8mb4

### ISSUE THREAD

**Issue by er314** [ryancramerdesign/FieldtypeTextUnique#2]
The module makes use of a varchar(255).
When used in an environment with InnoDB+utf8mb4, it fails at field creation.

For being compatible with InnoDB+utf8mb4, it should be retrofitted with the fixes, described here : 
https://github.com/ryancramerdesign/ProcessWire/issues/2020

**Comment by hiboudev** [comment]
I suppose this is the reason I can't update a Text field to a Text Unique field:
`ProcessWire: ProcessField: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'field_woltlabid_pwtmp' already exists`

Looks like this module is abandoned.

**Comment by er314** [comment]
In fact I raised this issue in 2018, 
but now the current version of the module is a complete rewrite (as stated in the README) which was performed in 2021.
Therefore the issue I raised in 2018 cannot be the root cause of the issue you are facing.
As a first conclusion, I think it's time that I close this issue :-) , so let's do it.
I think we can also say that the module is definitely NOT dead.
Then I'm replying on your issue #4 .
